app.js

```diff
@@ -17,6 +20,18 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 
+app.use(bodyParser.urlencoded({ extended: false }));
+// DMW session
+app.use(session({
+  secret: env.nodebb_session_secret,
+  resave: false,
+  saveUninitialized: true,
+  store:  new session.MemoryStore(),
```

Comment:
We should go to prod with MemoryStore. We should use proper database(redis or cassandra) with proper credentials/env variables.
only for local/dev testing we can go with MemoryStore

Reply:
Updated @vinukumar-vs